apps/src/templates/VisualizationOverlay.js

```diff
@@ -64,7 +64,7 @@ VisualizationOverlay.prototype.render = function (intoElement, nextProps) {
   var oldProps = $.extend({}, this.props_);
   $.extend(this.props_, nextProps);
 
-  if (this.props_.scale !== oldProps.scale) {
+  if (this.props_.scale !== oldProps.scale || !this.screenSpaceToAppSpaceTransform_) {
```

Comment:
I don't follow this particular change, but perhaps it is an existing bug that you're fixing?